tests/Stripe/HttpClient/CurlClientTest.php

```diff
@@ -352,4 +353,120 @@ public function testSetRequestStatusCallback()
             \Stripe\ApiRequestor::setHttpClient(null);
         }
     }
+
+    /**
+     * @after
+     */
+    public function tearDownTestServer()
+    {
+        $this->stopTestServer();
+    }
+
+    // This is a pretty flaky/uncertain way to try and get an
+    // http server to deliver the body in separate "chunks".
+    //
+    // It seems to work but feel free to just skip or delete
+    // if this flakes.
+    public function testExecuteRequestWithRetriesCallsWriteFunctionWithChunks()
```

Comment:
I wonder if there's something we can do to annotate these tests as more "heavy" or integration tests? I imagine their runtime is longer than standard unit tests. No real proposed change here, just commenting in case you know of something.

Reply:
PHPUnit has "test suites" https://phpunit.readthedocs.io/en/9.5/organizing-tests.html, but I don't think it's time for that yet.

I tuned the retry timeouts to bring the test suite down from 4.5s to 1.5s (still mildly distressing, it was sub 1s before)